tracing-core/src/span.rs

```diff
@@ -30,6 +30,22 @@ pub struct Record<'a> {
     values: &'a field::ValueSet<'a>,
 }
 
+/// TODO(eliza) docs
+#[derive(Debug)]
+pub struct Current {
+    inner: CurrentInner,
+}
+
+#[derive(Debug)]
+enum CurrentInner {
+    Current {
```

Comment:
Could this just straight up _hold_ a `Span`? That way there'd also be a convenient way to get a ref to a `Span` through this (which currently I don't think there is?).

Reply:
`Span` is defined in a different crate...

Reply:
Bah... Hmm... I guess what I'd like to see then is a wrapper around this in `tracing` that _actually_ gives you the current `Span`.

Reply:
Take another look at the diff in `tracing::Span`? :P

Reply:
Ohhhh, I missed that among the #147 changes! :tada: